scripts/update-config/LintPackage.ts

```diff
@@ -1,8 +1,8 @@
-import * as fs from "fs";
+import * as fs from "node:fs";
 import * as stringify from "json-stable-stringify";
-import * as path from "path";
+import * as path from "node:path";
 import { Configuration as Config, ILinterOptions, Linter, LintResult } from "tslint";
-import * as ts from "typescript";
+import ts from "typescript";
```

Comment:
I'm a little surprised by this; it's not a `*` import?

Reply:
I double checked:
```sh
$ echo '
import a from "typescript";
a.createSourceFile("input.ts", "");  // ✔️

import * as b from "typescript";
b.createSourceFile("input.ts", "");  // ❌
' > script.js
$ node script.js
file:///script.js:6
b.createSourceFile("input.ts", "");  // ❌
  ^

TypeError: b.createSourceFile is not a function
    at file:///script.js:6:3
    at ModuleJob.run (internal/modules/esm/module_job.js:145:37)
    at async Loader.import (internal/modules/esm/loader.js:182:24)
    at async Object.loadESM (internal/process/esm_loader.js:68:5)

Reply:
Oh, the tsconfig doesn't enable esModuleInterop, maybe that's what's going on. Maybe that's taboo in ESM mode?

Reply:
Hm, but you did test in js.